### DISCUSSION TITLE
Why does DarkReader apply a reddish tone?

### DISCUSSION THREAD

**Discussion by iUltimateLP** [darkreader/darkreader#7906] [General]
Hey there! I really love DarkReader and I'm using it for a lot of websites - but one thing I noticed it that it seems to apply a slightly reddish tone / color temperature to texts and other elements. Here's an example from Trello:

![image](https://user-images.githubusercontent.com/8491339/148738014-3c942584-ef5a-4d78-838a-3a85ff34e616.png)

And here's the same image hue-shifted with Photoshop:

![image](https://user-images.githubusercontent.com/8491339/148738230-37545833-5e76-4254-b01c-6e3344a5f69f.png)

This is with the sepia option off! Is there anything I can change? Thanks!

**Comment by Gusted** [comment]
Hi @iUltimateLP

I don't see the "reddish" tone. But it could be that you see this, this could might be because of your monitor settings. But I'd recommend if you give the new design a try, and check for the custom colors settings, and modify the colors to have a less reddish tone.

    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.

**Reply by iUltimateLP** [reply]
Thanks, with the new menu, it appears that DarkReader chose this:

![image](https://user-images.githubusercontent.com/8491339/148906292-8dcfd6b4-c49e-4ee2-817a-0cc661bb4b6b.png)

Setting it to #ffffff removes the reddish - so it might have been my monitor + something from DarkReader.

Thanks for the help, much appreciated!

**Comment by erlefloch** [comment]
I had the same issue, and this helped me a lot. Thanks @iUltimateLP  and @Gusted ! I think that this trick (enabling the new menu) should be mentioned in the help pages (https://darkreader.org/help/) , because probably other people might face the same issue.

**Comment by MrPeteH** [comment]
Clearly, it isn't color-neutral. It should be (unless the user sets sepia non-zero.)
This has nothing to do with monitors.
A handy Windows tool can be used to diagnose directly from RGB values on your screen (independent of monitor.) I've used the "Color Mania" app for many years. Has a color eye dropper that can show, pixel-by-pixel, exactly what color is on screen -- and includes direct hue translation.
